### PR TITLE
Added validation for invalid name chars (#2837)

### LINKED ISSUE #2837: Exams: Don't allow certain characters in first name / last name
These characters cause issues w/ Pearson. Short-term fix is to disallow them, we may revisit. Characters are: `>`,  `,` and `"`.

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### What are the relevant tickets?
Fixes #2837 

#### What's this PR do?
Adds validation to ensure `<`, `,` and `"` aren't allowed in names.

#### How should this be manually tested?
Try inputting the above chars into the romanized / non-romanized name fields

#### Where should the reviewer start?
`static/js/lib/validation/profile.js`

### CODE REVIEW

File: static/js/constants.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ export const DASHBOARD_FORMAT = 'MMM D, Y';
 export const DASHBOARD_MONTH_FORMAT = 'MM[/]YYYY';
 
 export const CP1252_REGEX = /^[\u0020-\u00FF]*$/;
+// greater-than, comma, quotation-mark
```
Comment (amir-qayyum-khan) [inline static/js/constants.js]: can we use `closing angle bracket` instead of `greater-than`. Because this is not related to programming,
 it is a text.

File: static/js/constants.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ export const DASHBOARD_FORMAT = 'MMM D, Y';
...
+export const INVALID_NAME_CHAR_REGEX = /[\u003E\u002C\u0022]/;
```
Comment (amir-qayyum-khan) [inline static/js/constants.js]: what about  `opening angular bracket <`. It allowed me to use it.

screen shot 2017-03-15 at 3 10 56 pm

Comment (amir-qayyum-khan) [inline static/js/constants.js]: why not `INVALID_NAME_CHARS_REGEX` because it is list if chars.

Comment (rhysyngsun) [inline static/js/constants.js]: only the characters listed in the original ticket are a concern

File: static/js/constants.js (hunk 1 above)
Comment (rhysyngsun) [inline static/js/constants.js]: It's a code comment which is programmer-facing so I'm inclined to leave it as-is because it's more succinct.

**Comment by amir-qayyum-khan** [comment]
👍
